### DISCUSSION TITLE
Prod and title are fine, but games say invalid file?

### DISCUSSION THREAD

**Discussion by Antihyro** [skyline-emu/skyline#816] [Q&A] [answered]
I've tried many prod and title files, and they all worked fine, but the games say invalid files, and yes, I have tried many ROMs so far I tried odyessy which I've heard news of working and it says invalid any help?

**Comment by PixelyIon** [comment, marked as answer]
It does seem to be an issue on your side rather than ours. Check that you've dumped your keys and games from your Switch correctly.

**Comment by ghost** [comment]
@Antihyro keys work fine 100%
